framework/contracts/account/manager/tests/common/mock_modules.rs

```diff
@@ -128,3 +128,58 @@ pub mod app_1 {
         );
     }
 }
+
+// standalone 1 have cw2
+pub mod standalone_1 {
+    pub use super::*;
+    pub const MOCK_STANDALONE_ID: &str = "tester:mock-standalone1";
+
+    #[cosmwasm_schema::cw_serde]
+    pub struct MockMsg;
+
+    pub fn mock_instantiate(
+        deps: ::cosmwasm_std::DepsMut,
+        env: ::cosmwasm_std::Env,
+        info: ::cosmwasm_std::MessageInfo,
+        msg: MockInitMsg,
+    ) -> cosmwasm_std::StdResult<cosmwasm_std::Response> {
+        cw2::set_contract_version(deps.storage, MOCK_STANDALONE_ID, V1);
+        Ok(cosmwasm_std::Response::new())
+    }
+
+    /// Execute entrypoint
+    pub fn mock_execute(
+        deps: ::cosmwasm_std::DepsMut,
+        env: ::cosmwasm_std::Env,
+        info: ::cosmwasm_std::MessageInfo,
+        msg: MockMsg,
+    ) -> cosmwasm_std::StdResult<cosmwasm_std::Response> {
+        Ok(cosmwasm_std::Response::new())
+    }
+
+    /// Query entrypoint
+    pub fn mock_query(
+        deps: ::cosmwasm_std::Deps,
+        env: ::cosmwasm_std::Env,
+        msg: MockMsg,
+    ) -> cosmwasm_std::StdResult<cosmwasm_std::Binary> {
+        Ok(cosmwasm_std::Binary::default())
+    }
+}
+
+// standalone 2 does not have cw2
+pub mod standalone_2 {
```

Comment:
Maybe the name can indicate that it doesn't have a cw2 instead of using a number?

Reply:
Sure, just didn't want them to stand out in this file. :eyes:
adapter_1, adapter_2, app_1, app_2, standalone_cw2, standalone_no_cw2